tests/foreman/cli/test_hammer.py

```diff
@@ -144,33 +136,32 @@ def test_positive_disable_hammer_defaults(target_sat):
 
     :expectedresults: Hammer --no-use-defaults product list should fail.
 
-    :CaseImportance: Critical
-
     :BZ: 1640644, 1368173
     """
-    default_org = make_org()
-    default_product_name = gen_string('alpha')
-    make_product({'name': default_product_name, 'organization-id': default_org['id']})
-    try:
-        Defaults.add({'param-name': 'organization_id', 'param-value': default_org['id']})
-        # list templates for BZ#1368173
-        result = target_sat.execute('hammer job-template list')
-        assert result.status == 0
-        # Verify --organization-id is not required to pass if defaults are set
-        result = target_sat.execute('hammer product list')
-        assert result.status == 0
-        # Verify product list fail without using defaults
-        result = target_sat.execute('hammer --no-use-defaults product list')
-        assert result.status != 0
-        assert default_product_name not in result.stdout
-        # Verify --organization-id is not required to pass if defaults are set
-        result = target_sat.execute('hammer --use-defaults product list')
-        assert result.status == 0
-        assert default_product_name in result.stdout
-    finally:
-        Defaults.delete({'param-name': 'organization_id'})
+    default_org = target_sat.cli_factory.make_org()
+    default_product = target_sat.cli_factory.make_product({'organization-id': default_org.id})
+
+    target_sat.cli.Defaults.add({'param-name': 'organization_id', 'param-value': default_org.id})
+    # list templates for BZ#1368173
+    result = target_sat.execute('hammer job-template list')
+    assert result.status == 0
```

Comment:
This does not simply confirms if there are any templates listed or empty list returned. So I would check the output and the list is not empty.

Reply:
Here we just would like to confirm that after specifying defaults param, hammer doesn't asks to pass org/orgid